### PR TITLE
Revision of releases page

### PR DESCRIPTION
**Pull request by DanielW1987**
Finally...

We can now see all upcoming releases. The Trello card contains a number of to-dos. For example, the right filter bar has no function yet.

### CODE REVIEW

**Comment by DanielW1987** [comment]
[Revision of releases page](https://trello.com/c/2aKFnPXh/91-revision-of-releases-page)

File: support/src/main/java/rocks/metaldetector/support/EnumPrettyPrinter.java [hunk 1]
```diff
@@ -3,10 +3,24 @@
 import org.apache.commons.text.WordUtils;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+
 @Service
 public class EnumPrettyPrinter {
 
+  private static final List<String> EXCEPTIONS = List.of(
```
Comment (hedgehog1833) [inline support/src/main/java/rocks/metaldetector/support/EnumPrettyPrinter.java:11]: The variable name really got me confused... but I don't have better ideas 🤷

**Comment by hedgehog1833** [comment]
Ah and I got a lot of 400 errors for the images but I guess that's my local database, haven't checked yet

**Comment by DanielW1987** [comment]
> Ah and I got a lot of 400 errors for the images but I guess that's my local database, haven't checked yet

I have the same error, it's your local database. Later we need something to use also images from local disc.

File: support/src/main/java/rocks/metaldetector/support/EnumPrettyPrinter.java (hunk 1 above)
Comment (DanielW1987) [reply support/src/main/java/rocks/metaldetector/support/EnumPrettyPrinter.java:11]: `EXCEMPTION`?

File: support/src/test/java/rocks/metaldetector/support/EnumPrettyPrinterTest.java [hunk 2]
```diff
@@ -34,6 +34,17 @@ void should_transform_fully_capitalized(String givenValue, String expectedValue)
     assertThat(result).isEqualTo(expectedValue);
   }
 
+  @ParameterizedTest(name = "Should not transform <{0}>")
```
[the remark this answers is not in the document]
Comment (DanielW1987) [reply support/src/test/java/rocks/metaldetector/support/EnumPrettyPrinterTest.java:37]: ...to be prepared to future enhancements of the exemption list.

File: support/src/main/java/rocks/metaldetector/support/EnumPrettyPrinter.java (hunk 1 above)
Comment (hedgehog1833) [reply support/src/main/java/rocks/metaldetector/support/EnumPrettyPrinter.java]: Yeah I guess that's better

**Review by hedgehog1833 — APPROVED** [review]
Niiice 🤘

**Pull request merged**